include/oneapi/dpl/pstl/parallel_backend_serial.h

```diff
@@ -71,6 +71,13 @@ __parallel_for(_ExecutionPolicy&&, _Index __first, _Index __last, _Fp __f)
     __f(__first, __last);
 }
 
+template <class _ExecutionPolicy, class _Index, class _Fp>
+void
+__parallel_for(oneapi::dpl::__internal::__tbb_backend_tag, _ExecutionPolicy&&, _Index __first, _Index __last, _Fp __f)
```

Comment:
It is not "tbb backend", it is a "serial backend" (a kind of trivial case of parallel backend).
So, it seems should be a `__serial_tag`.